shell/server/core.js

```diff
@@ -80,6 +80,10 @@ function hashSturdyRef(sturdyRef) {
   return Crypto.createHash("sha256").update(sturdyRef).digest("base64");
 }
 
+function generateSturdyRef() {
+  return Random.secret();
```

Comment:
Note that `Random.secret()` uses base64 which means it contains characters that are problematic for some contexts. I think Random.id(n) with a sufficiently-large n is what we want to use here.